common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -1298,3 +1298,32 @@ def test_lms_language_change(self):
             get_selected_option_text(language_selector),
             u'English'
         )
+
+
+@attr('shard_1')
```

Comment:
To make sure this runs in the correct context/shard,  use `@attr('a11y')` on the class, and not on the test case itself.